src/components/App/App.css

```diff
@@ -8,18 +8,16 @@ html {
   text-align: center;
 }
 
-.header-text {
-  font-size: 2.5rem;
-}
-
-.header-text-2 {
-  color: #9eeeff;
-}
-
-.App-header {
+.Aquestionnaire-header {
```

Comment:
Is this intended to be Aquestionnaire-header or did the A sneaked in?

Reply:
It snuck in! No clue how that happened, but it's fixed now.